src/main/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/messages.properties

```diff
@@ -21,7 +21,7 @@ javadoc.paragraph.misplaced.tag=<p> tag should be placed immediately before the
 javadoc.paragraph.redundant.paragraph=Redundant <p> tag.
 javadoc.paragraph.tag.after=Empty line should be followed by <p> tag on the next line.
 javadoc.parse.rule.error=Javadoc comment at column {0} has parse error. Details: {1} while parsing {2}
-javadoc.return.expected=Expected @return tag.
+javadoc.return.expected=@return tag should have description.
```

Comment:
`@return tag should be present and have description.`

Reply:
@ shashwatj07 , please do.
Sorry for several confusions.

Reply:
Done @romani